plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcClient.java

```diff
@@ -76,10 +76,10 @@ default boolean supportsAggregationPushdown(ConnectorSession session, JdbcTableH
 
     ConnectorSplitSource getSplits(ConnectorSession session, JdbcTableHandle tableHandle);
 
-    Connection getConnection(ConnectorSession session, JdbcSplit split)
+    Connection getConnectionForRead(ConnectorSession session, JdbcSplit split)
             throws SQLException;
 
-    Connection getConnection(ConnectorSession session, JdbcOutputTableHandle handle)
+    Connection getConnectionForWrite(ConnectorSession session, JdbcOutputTableHandle handle)
```

Comment:
That's redundant. `JdbcSplit` is for reading, `JdbcOutputTableHandle` is for writing.
Let's drop this rename.